### PR TITLE
Plotting

### LINKED ISSUE #142: Add Plottable Child to the Calculation endpoint

### PR DESCRIPTION
**Pull request by eatyourpeas**
This alters the Measurement Class to return all results also in plottable format. This allows clients to make a single API request for plottable data.
The plottable-child-data is still present
There is also a calculations endpoint which receives an array of measurement requests and returns an array of Measurement objects.
This should work with version 2.0 and above of the react client, and v4.0 and above of the charting library.

### CODE REVIEW

**Comment by eatyourpeas** [comment]
please note - if approved, the server version number will need updating to 0.04.

File: blueprints/uk_who_blueprint.py [hunk 1]
```diff
@@ -198,3 +198,76 @@ def uk_who_plottable_child_data():
...
+                sex: male
+                gestation_weeks: 40
+                gestation_days: 4
+
+      responses:
+        200:
+          description: "Centile calculation (single) according to the supplied data was returned"
+          content:
+            application/json:
+              schema: CalculationResponseSchema
```
Comment (pacharanero) [inline blueprints/uk_who_blueprint.py]: we need to add the plottable child bit to the schema

**Review by pacharanero — COMMENTED** [review]
Am I missing something? It looks like you've added the Calculations endpoint back in? Is that deliberate? We deprecated it months ago I thought? Unless I have it wrong

**Comment by eatyourpeas** [comment]
We did discuss it but in passing. I was thinking we would need it anyway in future when we start looking at SDS trends and so on. If you feel it is too vulnerable then comment it out and we can reenable in future,

**Comment by eatyourpeas** [comment]
Should now have the calculations commented out.

**Review by pacharanero — APPROVED** [review]

**Pull request merged**